tests/src/e2e/ledger_tests.rs

```diff
@@ -678,151 +677,6 @@ fn ledger_txs_and_queries() -> Result<()> {
     Ok(())
 }
 
```

Comment:
let's not delete this test, integration tests are easier to dbg and faster - this is just a simplified happy path test from https://github.com/anoma/namada/pull/1753 as a last safety net